parachain-tracer/src/prometheus.rs

```diff
@@ -253,6 +257,21 @@ impl PrometheusMetrics for Metrics {
 					.with_label_values(&[&para_string])
 					.inc();
 			}
+
+			for (index, address) in dispute_outcome.initiators.iter() {
+				metrics
+					.disputes_stats
+					.initiators
+					.with_label_values(&[&para_string, &index.to_string(), address])
```

Comment:
I think we need the session here as well, otherwise we will really get confused, because the validator Index is relevant only in the same session.